about.html

```diff
@@ -0,0 +1,135 @@
+                    <ul class="nav-menu">
+                        <li class="nav-menu-item"><a href="./index.html" class="nav-link">Home</a></li>
+                        <li class="nav-menu-item"><a href="#" class="nav-link">Program</a></li>
+                        <li class="nav-menu-item"><a href="#" class="nav-link">Presenters</a></li>
+                        <li class="nav-menu-item"><a href="#" class="nav-link">Partners</a></li>
+                    </ul>    
+                </div>
+                <i class="fas fa-bars"></i>
+              </div>
+            </nav>
+            <div class="mobile nav-line"></div>
+                    <div class="top-bar">
+            <ul>
+                <li><a href="#"><img class="social-icon" src="./images/facebook.png"></a></li>
+                <li><a href="https://www.facebook.com/mchafu"><img class="social-icon" src="./images/twitter.png"></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="https://thetjaypod.wordpress.com/">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+
+            </ul>
+        </div>
+            <div class="desk-nav">
+                <a class= "imagea" href="./index.html"><img class="logo" src="./images/Shakys-logo.jpeg"></a>
+                <ul class="desk-nav-menu">
+                    <li class="desk-nav-menu-item"><a class="desk-nav-link" href="./index.html">Home</a></li>
+                    <li class="desk-nav-menu-item"><a class="desk-nav-link" href="#">Program</a></li>
+                    <li class="desk-nav-menu-item"><a class="desk-nav-link" href="#">Speakers</a></li>
+                    <li class="desk-nav-menu-item"><a class="desk-nav-link" href="#">Partners</a></li>
+                </ul>
+                <div class="register-now">Register Now</div>
+            </div>
+    
+          </header>
+    <div class="about-page-info">
+        <div class="headings">
+            <h2 class="quote">"Good Food Unites Hearts, Nourishes Souls."</h2>
+            <h1 class="topic">SHAKY'S ANNUAL COOKOUT 2023</h1> 
+            </div>
+        
+        <div class="home-message">
+            <p class="welcome-message">
+                Shaky's Cookout is an annual event that brings together people from various industries to network, socialize, and enjoy delicious food. It provides an opportunity for professionals to come together, exchange ideas, and build relationships in a relaxed and fun environment. 
+            </p>
+        </div>
+        <div>
+            <p class="further-info">For further information about the cookout contact us at shakysconference2023@shakys.co.ke</p>
+        </div>
+    </div>
+    </section>
+    <section class="section-2">
+        <h2 class="main-program-heading">
+           Shaky's Conference
+        </h2>
+        <div class="red-line1">
+
+        </div>
+        <div class="about-shakys">
+            Awarding the best chefs in the country since 2021.   
```

Comment:
- [ ] please the popup menu should fit full width and height
- [optional] the menu popup is meant for navigation, and it will lose its role if when it comes up a user is still able to  scroll through the application without using it. Kindly make it static so unless it is close it must be used to navigate.
![Screenshot (816)_LI](https://user-images.githubusercontent.com/65298048/219626770-78937fed-94a3-47fd-b87f-9e826665230e.jpg)